doc/api/http.md

```diff
@@ -587,9 +587,11 @@ added: v1.6.0
 -->
 
 * `name` {string}
-* Returns: {string}
+* Returns: {string, string[]} - type as set via `setHeader()`
```

Comment:
The `type as set via` part seems out of place. If the information is critical, it can be included in the text below.

Use a `|` to separate the two possible return types, not a string. (This is to be consistent with our documentation elsewhere.)

Reply:
@Trott "not a string" -> "not a comma"?

Reply:
@vsemozhetbyt Yes, thanks, updated!